guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning in IPv4 Network 
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
+
+.Procedure
+
+. On {Project}, enable the `foreman-proxy-http` and `foreman-proxy-httpboot` plug-ins. Disable system checks with the `--disable-system-checks`  and `--skip-checks-i-know-better` options.
```

Comment:
> Disable system checks with the `--disable-system-checks`  and `--skip-checks-i-know-better` options.

I can't remember why I had these, we need confirmation from QE if it works without these. I believe these were some temporary workarounds for my setup.

Reply:
Probably in real setups users won't need them, I'll remove